### PR TITLE
UI Design and Themeing

### PR DESCRIPTION
**Pull request by Xeonus**
- Port ZEN UI Themeing
- Upgrade providers and dependencies
- Upgrade pages to correctly parse theme elements
- Hero landing page
- Sidebar navigation (WIP)

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -1,11 +1,64 @@
...
+                        size="4xl"
+                        variant="special"
+                        fontWeight="bold"
+                    >
+                        Welcome to Balancer Ops Tooling
+                    </Heading>
+                    <Text fontSize="xl" variant="secondary">
+                        Streamlining Balancer DAO operations with our powerful suite of tools
+                    </Text>
+                    {!showOptions ? (
```
Comment (Hyferion) [inline app/page.tsx:34]: I would show the options directly. The click is kinda unessacary IMO. But we can do this later. Just noting

File: lib/modules/components/Sidebar.tsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+                alignItems={navSize == "small" ? "center" : "flex-start"}
+                mb={4}
+            >
+                <Divider display={navSize == "small" ? "none" : "flex"} />
+                <Flex
+                    mt={4}
+                    align="center"
+                    transition="all 0.3s ease"
+                >
+                    <Avatar size="sm" />
```
Comment (Hyferion) [inline lib/modules/components/Sidebar.tsx:82]: Could probs be removed

File: app/payload-builder/ccip-bridge/page.tsx
Comment (Hyferion) [inline app/payload-builder/ccip-bridge/page.tsx:1]: We should build a hook that queries the address book based on parameters so we can reuse

**Comment by Hyferion** [comment]
LGTM. Ready to merge.

**Review by Hyferion — APPROVED** [review]

File: app/payload-builder/ccip-bridge/page.tsx
Comment (Xeonus) [reply app/payload-builder/ccip-bridge/page.tsx:1]: Have a TODO in the relevant sections. First step was to port everything over

**Pull request merged**